app/views/layouts/application.html.erb

```diff
@@ -46,7 +45,7 @@
         <!-- Add styling to center -->
         <p>Made with love by the #April2017 <a class="footer-link" href="http://makersacademy.com">Makers Academy</a> cohort</p>
       </div>
-
     </footer>
-  </body>
+    <%= javascript_include_tag 'application' %>
```

Comment:
@sulaimancode is right and has a solution to this. Talk to him about how to use it.